File: CSharp/ShopTest/Items/ItemTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Xunit;

namespace ShopTest.Items {
    using Shop.Items;
    public class ItemTest {
        [Fact]
        public void NameAndValue_MatchGiven() {
            Item i = new Item("Red Pants", 60);

            Assert.Equal("Red Pants", i.Name);
            Assert.Equal(60, i.Value);
        }
    }
}

File: Java/TddShop/TddShopTest/src/Item/ItemTest.java
package Item;

import org.junit.Assert;
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class ItemTest {
    @Test
    public void NameAndValue_EqualGiven(){
        Item i = new Item("Black Shirt", 50);
        Assert.assertEquals("Black Shirt", i.Name());
        Assert.assertEquals(50, i.Value());
    }
}
